src/c++/perf_analyzer/load_manager.cc

```diff
@@ -512,6 +518,9 @@ LoadManager::PrepareInfer(InferContext* ctx)
 cb::Error
 LoadManager::PrepareSharedMemoryInfer(InferContext* ctx)
 {
+  // FIXME: TMA-765 - Shared memory mode does not support optional inputs,
```

Comment:
Can you move the comment just before L544? 

> ctx->valid_inputs_.push_back(infer_input);

Reply:
Ooh, gotcha--I misunderstood what you meant [here](https://github.com/triton-inference-server/client/pull/127#discussion_r924804662) when you said `at the top`. Moved 👍